Cargo.toml

```diff
@@ -190,3 +190,6 @@ vendored = [
   "rusqlite/bundled-sqlcipher-vendored-openssl",
   "reqwest/native-tls-vendored"
 ]
+
+[lints.rust]
+unexpected_cfgs = { level = "warn", check-cfg = ['cfg(fuzzing)'] }
```

Comment:
Can we add this configuration instead somehow?

Reply:
Looks like this can be converted to a feature:
```
[features]
...
fuzzing = []
```

Reply:
At least now i don't see in the bolero documentation/examples that it's needed

Reply:
And it supports `--features` option

Reply:
Indeed, but it's strange that i don't see anything with `fuzzing` configuration in their examples. Ok, let's keep this as is